llvm/lib/Target/RISCV/RISCVInstrInfoXSf.td

```diff
@@ -553,29 +560,40 @@ class GetFTypeInfo<int Sew> {
 }
 
 multiclass VPatVMACC<string intrinsic, string instruction, string kind,
-                     list<VTypeInfoToWide> info_pairs, ValueType vec_m1> {
+                     list<VTypeInfoToWide> info_pairs, ValueType vec_m1,
+                     bit lmul_follows_vd = 0> {
```

Comment:
lmul doesn't follow vd for any of these instructions. It always follows vs2, but one of the instructions has the same lmul for vs2 and vd.

Reply:
Oh, that's right.